piker/data/_ahab.py

```diff
@@ -143,27 +143,37 @@ async def process_logs_until(
                 if not entry:
                     continue
 
+                entry = entry.strip()
                 try:
-                    record = json.loads(entry.strip())
+                    record = json.loads(entry)
+
+                    if 'msg' in record:
+                        msg = record['msg']
+                    elif 'message' in record:
+                        msg = record['message']
+                    else:
+                        raise KeyError('Unexpected log format')
+
+                    level = record['level']
+
                 except json.JSONDecodeError:
-                    if 'Error' in entry:
-                        raise RuntimeError(entry)
-                    raise
+                    # if 'Error' in entry:
+                    #     raise RuntimeError(entry)
+                    # raise
+                    msg = entry
+                    level = 'error'
 
-                msg = record['msg']
-                level = record['level']
                 if msg and entry not in seen_so_far:
                     seen_so_far.add(entry)
                     if bp_on_msg:
                         await tractor.breakpoint()
 
-                    getattr(log, level, log.error)(f'{msg}')
+                    getattr(log, level.lower(), log.error)(f'{msg}')
 
-                    # print(f'level: {level}')
-                    if level in ('error', 'fatal'):
+                    if level == 'fatal':
                         raise ApplicationLogError(msg)
 
-                if patt in msg:
+                if patt_matcher(msg):
```

Comment:
I like this predicate usage but we should maybe put a comment near the type sig at the top of this routine mentioning that this is a predicate func for matching log msgs emitted by the underlying containerized app yah?